test/gpu/native/studies/shoc/sort.compopts

```diff
@@ -0,0 +1 @@
+result.chpl --main-module=sort --no-checks
```

Comment:
I just took put that in there to get rid of the warning that is generated. I can take out the `--no-checks` and modify the `.good` file appropriately if you'd like.

Reply:
I would. It's ugly but it's what we do for other tests.  Again my main concern is one day we change the behavior so that `--no-checks` is no longer implied and this test unnecessarily keeps that flag in.